### PR TITLE
mokuro: 0.1.8 -> 0.2.1

### PR DESCRIPTION
**Pull request by daspositioned**
## Description of changes

Changelogs:
https://github.com/kha-white/mokuro/releases/tag/v0.2.1
https://github.com/kha-white/mokuro/releases/tag/v0.2.0

I'm not entirely sure if this would be considered a breaking change since it fundamentally changes how the app works (it no longer generates .html files but .mokuro files, which have to be loaded in an external application). However, this change is clearly stated when running this new version.

image

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/mo/mokuro/package.nix [hunk 1]
```diff
@@ -6,26 +6,26 @@
...
-    substituteInPlace setup.py \
+    substituteInPlace pyproject.toml \
         --replace-fail 'opencv-python' 'opencv'
   '';
 
 
   pythonRelaxDeps = [ "torchvision" ];
 
-  build-system = with python3Packages; [ setuptools ];
+  build-system = with python3Packages; [ setuptools_scm ];
```
Comment (AndersonTorres) [inline pkgs/by-name/mo/mokuro/package.nix:28]: `setuptools_scm` was not declared. Fix.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Comment by eclairevoyant** [comment]
> I'm not entirely sure if this would be considered a breaking change since it fundamentally changes how the app works

Then by definition yes, it's breaking, not backportable.

File: pkgs/by-name/mo/mokuro/package.nix (hunk 1 above)
Comment (dlsf) [reply pkgs/by-name/mo/mokuro/package.nix]: You're right, I even tested this... No idea what went wrong there.

**Comment by dlsf** [comment]
@eclairevoyant I see, thank you for your information. But is that relevant here? The package is only in unstable right now. If it is relevant, is there anything I need to do?

**Comment by eclairevoyant** [comment]
> But is that relevant here? The package is only in unstable right now.

I assumed you mentioned breaking changes with the intent of possibly backporting; if it's not in stable then not relevant here.

**Review by AndersonTorres — APPROVED** [review]

**Comment by eclairevoyant** [comment]
Thanks for your contribution!

**Pull request merged**